sopel_modules/github/webhook.py

```diff
@@ -102,16 +106,32 @@ def stop(self):
 
 
 def get_targets(repo):
-    conn = sopel_instance.db.connect()
-    c = conn.cursor()
-    c.execute('SELECT * FROM gh_hooks WHERE repo_name = ? AND enabled = 1', (repo.lower(), ))
-    return c.fetchall()
+    # Catch any errors connecting to database
+    try:
+        engine = sopel_instance.db.connect()
+    except OperationalError:
+        print("OperationalError: Unable to connect to database.")
+        raise
+
+    session = scoped_session(sessionmaker(bind=engine))
```

Comment:
Sometimes you use `engine`, sometimes `engine.engine`. Are you sure about this code?

Reply:
Weirdly it works both ways lol